Configuration/Skimming/python/Skims_PA_cff.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
It looks like the import of EventContentHeavyIons_cff is breaking the replay with 8_0_23.
Load of this files leads to override of all the standard EventContent (AOD, RAW, Reco, ) with their HI extensions.

Specifically, one of the overrides comes from RecoHiEgamma_EventContent_cff from RecoHiEgammaAOD, which has some ill-formed ```keep floatedmValueMap_*_*_*```.
I would be still OK, if we were not running miniAOD: regular reco config setup purges the unnecessary ak8PFJetsCHSFilteredMass and its upstream ak8PFJetsCHSFiltered from the configuration when creating unscheduled configuration; but then miniAOD loads it back again in PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py as   ```process.load("RecoJets.JetProducers.ak8PFJetsCHS_groomingValueMaps_cfi")``` and there it loads only ak8PFJetsCHSFilteredMass and there is no downstream consumers of it in miniAOD and in this case this module should not run ... except that you have now inadvertently added ```keep floatedmValueMap_*_*_*``` which makes ak8PFJetsCHSFilteredMass runnable.
We didn't hit any of these fun parts with ```keep *``` in 2015 PbPb run because we did not run miniAOD.

If we are running the standard pp sequence with some small additions and edits of modules for pA needs, the FEVT event content has to be also the standard EventContent_cff.
So, please change the above 
```
from Configuration.EventContent.EventContentHeavyIons_cff import FEVTEventContent
```
to 
```
from Configuration.EventContent.EventContent_cff import FEVTEventContent
```